subsys/event_manager/Kconfig

```diff
@@ -27,14 +27,26 @@ config EVENT_MANAGER_SHOW_EVENT_HANDLERS
 	help
 	  This option controls if event handlers are printed to console.
 
+config EVENT_MANAGER_TRACE_EVENT_DATA
+	bool "Enables tracing information"
```

Comment:
I think we should select it when we enable profiling event manager events (that would simplify configuration)